### PR TITLE
[11.0] Fix No update cron data purchase_batch_invoicing

### PR DESCRIPTION
**Pull request by RodrigoBM**
Fix No update cron data purchase_batch_invoicing

### CODE REVIEW

File: purchase_batch_invoicing/data/ir_cron_data.xml [hunk 1]
```diff
@@ -3,15 +3,15 @@
      License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
 
 <odoo>
```
Comment (pedrobaeza) [inline purchase_batch_invoicing/data/ir_cron_data.xml]: Put for reducing diff:
```suggestion
<odoo noupdate="1">
```

**Review by pedrobaeza — APPROVED** [review]

File: purchase_batch_invoicing/data/ir_cron_data.xml (hunk 1 above)
Comment (RodrigoBM) [reply purchase_batch_invoicing/data/ir_cron_data.xml]: Done

**Comment by RodrigoBM** [comment]
Hi @pedrobaeza 

I have changed the code with your recommendations, it is possible that you accept it.

thank you very much

**Review by pedrobaeza — APPROVED** [review]

**Comment by RodrigoBM** [comment]
Thanks @pedrobaeza @dreispt

**Pull request merged**